buildSrc/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -0,0 +1,67 @@
+object Deps {
+
+    private const val kotlinVersion = "1.3.50"
+    private const val coroutinesVersion = "1.3.2"
+
+    val kotlin = Kotlin
+    val kotlinx = Kotlinx
+    val android = Android
+    val rxjava2 = "io.reactivex.rxjava2:rxjava:2.2.7"
+    val rxjava3 = "io.reactivex.rxjava3:rxjava:3.0.0-RC3"
+    val picasso = "com.squareup.picasso:picasso:2.71828"
+    val bintray = "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4"
+    val detekt = "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.1.0"
+
+    object Kotlin {
+        val stdlib = Stdlib()
+        val test = Test
+        val plugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+
+        class Stdlib(
+            private val name: String = "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+        ) : CharSequence by name {
+            val jdk7 = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+            val js = "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlinVersion"
+            val common = "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion"
+
+            override fun toString(): String = name
```

Comment:
Why here we have `name` and `toString` and `CharSequence` and don't have it for e.g. `Test`?

Reply:
You can use both `Deps.kotlin.sdtlib` and `Deps.kotlin.stlib.common`.
`implementation` call accepts `CharSequence`, so to achieve behaviour above, I implemented `CharSequence` interface on `Stdlib` class and provided `name` property for interface delegation.
Otherwise you need to introduce different field on `Stdlib` object for `org.jetbrains.kotlin:kotlin-stdlib` and API won't be so cool (like `Deps.kotlin.sdtlib.default`).

Reply:
`toString()` method is not delegated to `name`, so I have to implement it manually. Gradle calls `toString()` on this object somewhere inside.